homeassistant/components/philips_js/strings.json

```diff
@@ -6,6 +6,13 @@
           "host": "[%key:common::config_flow::data::host%]",
           "api_version": "API Version"
         }
+      },
+      "pair": {
+        "title": "Pair",
+        "description": "Enter the [%key:common::config_flow::data::pin%] displayed on your TV",
```

Comment:
We should not combine translations like this. Please remove the common key in this case.

Reply:
Done